woocommerce/woocommerce-gutenberg-products-block
# Remove styling from sale badge

## What

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/10862

Removes the parent styling of the image from sale badge block.

## Why



User customizing the styling such as padding/margin and border radius should not affect how the sale badge looks.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a post and add the Single Product Block.
2. Select a product that has a sale price.
3. Turn on Show On-Sale Badge.
4. Go to Styles in the inspector controls and apply padding/margins and border radius.
5. Ensure it does not affect the Sale Badge. It should only affect the product image.
6. Save and check the frontend matches what you see in the editor.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Single Product Block: Prevent image styling to affect the on-sale badge.
